php-packages/phpstan/src/Attributes/ModelDateAttributesExtension.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\PHPStan\Attributes;
+
+use Carbon\Carbon;
+use Flarum\PHPStan\Extender\MethodCall;
+use Flarum\PHPStan\Extender\Resolver;
+use PHPStan\Reflection\ClassReflection;
+use PHPStan\Reflection\PropertiesClassReflectionExtension;
+use PHPStan\Reflection\PropertyReflection;
+use PHPStan\Type\NullType;
+use PHPStan\Type\ObjectType;
+use PHPStan\Type\UnionType;
+
+class ModelDateAttributesExtension implements PropertiesClassReflectionExtension
```

Comment:
There's a lot of code repetition; maybe we should consider some abstract class on top of `PropertiesClassReflectionExtension` for finding extenders?

Reply:
We can further iterate on this 👍🏼